### PR TITLE
Bug 1974973: Using dynamic release for P/Z 

### PR DESCRIPTION
**Pull request by deepsm007**
cc @alvaroaleman @stevekuznetsov 

Signed-off-by: Deep Mistry

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__s390x-nightly-4.9.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    tag: libvirt-installer
+  upi-installer:
+    name: "4.9"
+    namespace: ocp
+    tag: upi-installer
+  vsphere-csi-driver-operator-test:
+    name: "4.9"
+    namespace: ocp
+    tag: vsphere-csi-driver-operator-test
+releases:
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/release/openshift-release-master__s390x-nightly-4.9.yaml:62]: Define the s390x releases with names that are *not* `latest` and `initial`, then use [dependency overrides](https://docs.ci.openshift.org/docs/architecture/ci-operator/#dependency-overrides) to tell the installer to use the s390x releases for installation

Comment (deepsm007) [reply ci-operator/config/openshift/release/openshift-release-master__s390x-nightly-4.9.yaml:62]: I tried updating based on dependency overrides but do you think it also requires anything else to resolve ?

**Comment by jaypoulz** [comment]
The only question I have, and probably @r4f4 knows best, is how do you make sure we pick up the P/Z initial/lastest in the case of the upgrade tests?

**Comment by r4f4** [comment]
On Tue, 20 Jul 2021, 21:02 Jeremy Poulin, ***@***.***> wrote:

> The only question I have, and probably @r4f4 <https://github.com/r4f4>
> knows best, is how do you make sure we pick up the P/Z initial/lastest in
> the case of the upgrade tests?
>
In the case of templates, we used our own MULTIARCH_RELEASE_IMAGE_INITIAL
to override the x86 RELEASE_IMAGE_INITIAL and then we used a variable
INSTALL_INITIAL_RELEASE to check if we should install from it instead of
latest.

In step-registry, they use OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
pointing to "release:initial" as the base install version
and OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE pointing to "release:latest"
as the upgrade target.

So as long as we have multiarch releases to refer to, we can use them as
content for those variables. However, we might want to do that at the test
level using the dependency override Steve mentioned so we don't have to
create one workflow for power and other for Z.

>
>

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**